modules/cnf-managing-device-interrupt-processing-for-guaranteed-pod-isolated-cpus.adoc

```diff
@@ -35,7 +57,7 @@ spec:
 [id="disabling_interrupt_processing_for_individual_pods_{context}"]
 == Disabling interrupt processing for individual pods
 
-To disable interrupt processing for individual pods, ensure that `globallyDisableIrqLoadBalancing` is set to `false` in the performance profile. Then, in the pod specification, set the `irq-load-balancing.crio.io` and `cpu-load-balancing.crio.io` pod annotations to `disable`. An example pod specification snippet that illustrates this is below:
+To disable interrupt processing for individual pods, ensure that `globallyDisableIrqLoadBalancing` is set to `false` in the performance profile. Then, in the pod specification, set the `irq-load-balancing.crio.io` pod annotation to `disable`. An example pod specification snippet that illustrates this is below:
```

Comment:
```suggestion
To disable interrupt processing for individual pods, ensure that the`globallyDisableIrqLoadBalancing` annotation is set to `false` in the performance profile. Then, in the pod specification, set the `irq-load-balancing.crio.io` pod annotation to `disable`. The following pod specification contains this annotation:
```

I'm not sure what the difference between the "performance profile" and the "pod specification" are in this example. Should this YAML snippet also contain `globallyDisableIrqLoadBalancing`, or does it go somewhere else?

Reply:
> Generally, specify the keyword type after the keyword.

Reply:
I had that same question, then I looked at the previously defined  `PerformanceProfile` so I felt it was OK not to repeat/add this here as the guidance states create a performance profile where `globallyDisableIrqLoadBalancing` is set to false

Reply:
I'm more curious exactly where it should go in the performance profile.

Reply:
It would be like previous example, 
```
apiVersion: performance.openshift.io/v2
kind: PerformanceProfile
metadata:
  name: manual
spec:
  globallyDisableIrqLoadBalancing: false
``` 

We have a pod spec and performance profile spec and from what I can see you reference the performance profile spec when you are setting up the pod spec. 
```
spec:
    runtimeClassName: performance-<profile_name>
```